lib/wm_util.ml

```diff
@@ -111,11 +111,71 @@ module MediaType = struct
     loop ranges
 end
 
-module Date = struct
-  open CalendarLib
+module type Date_sig = sig
```

Comment:
`Date_sig` -> `Date`. Signatures and modules have different namespaces, so there's no chance they'll conflict.

Reply:
Well, they constantly conflict in my brain. That's why I prefer the explicit style. It makes it more readable. But of course I will adapt to any style you prefer.